### PR TITLE
Remove .jsonlines hack from simple swing store

### LINKED ISSUE #3134: remove "jsonlines" persistent mode from swing-store-simple
We're planning to remove the (hideously inefficient) "jsonlines" persistence code from swing-store-simple, rather than add persistent transcript/"streamStore" support to it.

However, @michaelfig points out that cosmic-swingset needs it a little bit longer, until #3131 is complete and cosmic-swingset is moved fully over to swing-store-lmdb (removing the fallback to swing-store-simple, now that we made LMDB work under WSL).

So the task is to remove this code, but not until he gives us the go-ahead.

> Do you need more than one value? Maybe this could just go into a single one-line file.

I can easily implement what I need.  Just please don't remove the jsonlines implementation before I have a chance to do that.  The unit tests for `packages/agoric-cli` don't exercise this requirement (though the `packages/solo` unit tests do).  I'll fix that omission ASAP.

_Originally posted by @michaelfig in https://github.com/Agoric/agoric-sdk/pull/3122#discussion_r635567189_

### PR DESCRIPTION
**Pull request by FUDCo**
Closes #3134

### CODE REVIEW

**Review by warner — APPROVED** [review]
changes look good to me

File: packages/solo/src/access-token.js
Comment (FUDCo) [reply packages/solo/src/access-token.js]: This looks to me like it's using the SwingStore API but its own database (i.e., not the swingset database), which may be overkill but aside from (modest) overhead seems as good a way to store anything given the prior that you are going to store it.  But Michael can tell us whether or not I'm right about that.

Comment (michaelfig) [reply packages/solo/src/access-token.js]: @FUDCo is correct.  This use of `swing-store-simple` had no relationship to SwingSet.

It was not overkill when the database was a simple jsonlines file.  I really don't want to make it LMDB because that introduces portability concerns for what should just be a text file.

Quoting myself:
> I can easily implement what I need. Just please don't remove the jsonlines implementation before I have a chance to do that.

Here is a sample of the current contents of the file:

```js
soil:20210528 michael$ cat ~/.agoric/swingset-kernel-state.jsonlines 
["accessToken/8000","4S0..."]
["accessToken/7999","p0Q..."]
["accessToken/8001","Ifr..."]
["accessToken/9000","ArB..."]
["accessToken/6892","7P-..."]
```

**Review by michaelfig — CHANGES_REQUESTED** [review]
I'm not yet ready for swing-store-simple to vanish.  I will create a PR stacked on this one, but until then, please do not merge.

**Comment by FUDCo** [comment]
> I'm not yet ready for swing-store-simple to vanish. I will create a PR stacked on this one, but until then, please do not merge.

There's nothing else that's gated by this, so just give a yell when you're clear of the blast zone.

**Comment by michaelfig** [comment]
> There's nothing else that's gated by this, so just give a yell when you're clear of the blast zone.

Can you please review #3216 ?  Merging that into this PR is necessary before I can give a yell.

**Review by michaelfig — APPROVED** [review]

**Comment by michaelfig** [comment]
> just give a yell when you're clear of the blast zone.

/me yells!

**Pull request merged**
